### ISSUE TITLE
Plugin fails with DuplicateTaskException when used on Multiplatform projects with both JVM and Android targets

### ISSUE THREAD

**Issue by garyp** [Kotlin/binary-compatibility-validator#47]
I have a Kotlin Multiplatform project that defines both `jvm` and `android` targets (this is a snippet of the gradle config, I can include the full one if it's helpful):

```kt
plugins {
    kotlin("multiplatform")
    id("com.android.library")
    id("binary-compatibility-validator")
}

kotlin {
    android {
        publishLibraryVariants("release")
    }
    jvm()
    ...
}
```

If I apply v0.4.0 of the `binary-compatibility-validator` gradle plugin, I get the following error:

```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':runtime'.
> org.gradle.api.internal.tasks.DefaultTaskContainer$DuplicateTaskException: Cannot add task 'apiBuild' as a task with that name already exists.
```

This error didn't happen with v0.2.4 of the plugin. Looking at https://github.com/Kotlin/binary-compatibility-validator/blob/master/src/main/kotlin/BinaryCompatibilityValidatorPlugin.kt#L85, it seems that the plugin tries to create a task named `apiDump` for every `jvm` or `android` compilation in the project. This works fine if the project is configured with a `jvm` target _or_ an `android` target, but not if the project is configured with both targets.

Related to #38

**Comment by qwwdfsad** [comment]
Could you please provide a full snippet of the Gradle project or even the self-contained project itself?

**Comment by CherryPerry** [comment]
You can try to use 0.5.0 version in [Reaktive](https://github.com/badoo/Reaktive), we have the same issue. Update the version [here](https://github.com/badoo/Reaktive/blob/master/includedBuild/dependencies/src/main/kotlin/com/badoo/reaktive/dependencies/Deps.kt#L48) and try to `apiDump`. If I disable Android target [here](https://github.com/badoo/Reaktive/blob/master/includedBuild/gradleConfiguration/src/main/kotlin/com/badoo/reaktive/configuration/MppConfigurationPlugin.kt#L37) it works as expected.

**Comment by garyp** [comment]
> Could you please provide a full snippet of the Gradle project or even the self-contained project itself?

The project is https://github.com/streem/pbandk. If you checkout `master` and update `binaryCompatibilityValidatorGradlePlugin` in `buildSrc/src/main/kotlin/Versions.kt` you should be able to reproduce the issue.

**Comment by RinOrz** [comment]
@qwwdfsad Please fix this problem, when you use the same error with the Android target, version: 0.7.1

**Comment by qwwdfsad** [comment]
Should be fixed in 0.8.0-RC.
